### ISSUE TITLE
does not change setting at start //ubuntu 20

### ISSUE THREAD

**Issue by mchuc** [polychromatic/polychromatic#267]
I think that after logging on/off the computer, the application should automatically set the last color profile;
e.g. I have several users at home with me and I would like to have my own settings after logging in right away and not to switch manually

**Comment by lah7** [comment]
Agreed. This is something I'll address in the upcoming v1.0.0 update.

**Comment by mchuc** [comment]
Welcome  polychromatic/polychromatic! 


You are invited to access PLAFYA's Customer Self Service Portal 


 By accessing this portal, you can track your requests online,

access knowledge base and join the community forums.
ACCEPT THE INVITATION



 




Your user name is reply@reply.github.com | You can also click here to accept the invitation 





 



Thank you, PLAFYA Administrator






 








This email has been sent to you&#x28;reply&#x2b;ald3pf7sxwa5foy7cpoiizn5dyoctevbnhhcoucxxq&#x40;reply.github.com&#x29; by wizyty&#x40;afya.pl from PLAFYA. Please contact the sender&#x28;wizyty&#x40;afya.pl&#x29; for any clarifications.

**Comment by mchuc** [comment]
thank You, Master!

**Comment by gemenerik** [comment]
So it is correct behavior that the settings are not applied automatically on reboot/relog?

**Comment by mchuc** [comment]
Praktyka Lekarskia AFYA, https://afya.pl 
Witaj  polychromatic/polychromatic! 


Otrzymaliśmy Państwa wiadomość e-mail,
 a nasz System informatyczny automatycznie nadał jej numer. 

W dalszym kontakcie - prosimy nie zmieniać tytułu wiadomości
 i posługiwać się właśnie tym numerem


Nie oznacza to, że wiadomość została przeczytana, a jedynie dostarczona.
Odpowiadamy zwykle w ciągu 4 dni roboczych.

Tę wiadomość otrzymacie Państwo jednokrotnie (nie wyślemy jej ponownie nawet przy prawidłowym dostarczeniu kolejnych wiadomości e-mail), dlatego  zapraszamy do odwiedzenia Naszego Centrum Pomocy  Customer Self Service Portal


 Wchodząc tam, możecie śledzić swoje wiadomości i odpowiedzi na nie w trybie rzeczywistym, uzyskacie także dostęp do bazy wiedzy i będziecie mogli dołączyć do forów społecznościowych.


ZAAKCEPTUJ ZAPROSZENIE



 
W każdym przypadku wątpliwości 
zapraszamy do kontaktu z naszą rejestracją pod numerem telefonu
+48 71 88 08 555


Państwa nazwa Użytkownika to:  reply@reply.github.com | Możecie również kliknąć tutaj  kliknij tutaj, aby zaakceptować to zaproszenie automatycznie. 





 



Dziękujemy za kontakt. Zespół AFYA.pl






 








This email has been sent to you&#x28;reply&#x2b;ald3pf6j5ujolopdcvprzh55mm6fzevbnhhcoucxxq&#x40;reply.github.com&#x29; by wizyty&#x40;afya.pl from Praktyka Lekarska AFYA - POMOC. Please contact the sender&#x28;wizyty&#x40;afya.pl&#x29; for any clarifications.

**Comment by lah7** [comment]
Not yet, this feature will be part of the **Triggers** tab/feature, which is not yet implemented.

For now, the workaround is to create a Python script (interfacing directly to OpenRazer) and add it to your desktop environment's "Start-up Applications".

**Comment by mchuc** [comment]
Praktyka Lekarskia AFYA, https://afya.pl 
Witaj  polychromatic/polychromatic! 


Otrzymaliśmy Państwa wiadomość e-mail,
 a nasz System informatyczny automatycznie nadał jej numer. 

W dalszym kontakcie - prosimy nie zmieniać tytułu wiadomości
 i posługiwać się właśnie tym numerem


Nie oznacza to, że wiadomość została przeczytana, a jedynie dostarczona.
Odpowiadamy zwykle w ciągu 4 dni roboczych.

Tę wiadomość otrzymacie Państwo jednokrotnie (nie wyślemy jej ponownie nawet przy prawidłowym dostarczeniu kolejnych wiadomości e-mail), dlatego  zapraszamy do odwiedzenia Naszego Centrum Pomocy  Customer Self Service Portal


 Wchodząc tam, możecie śledzić swoje wiadomości i odpowiedzi na nie w trybie rzeczywistym, uzyskacie także dostęp do bazy wiedzy i będziecie mogli dołączyć do forów społecznościowych.


ZAAKCEPTUJ ZAPROSZENIE



 
W każdym przypadku wątpliwości 
zapraszamy do kontaktu z naszą rejestracją pod numerem telefonu
+48 71 88 08 555


Państwa nazwa Użytkownika to:  reply@reply.github.com | Możecie również kliknąć tutaj  kliknij tutaj, aby zaakceptować to zaproszenie automatycznie. 





 



Dziękujemy za kontakt. Zespół AFYA.pl






 








This email has been sent to you&#x28;reply&#x2b;ald3pf7q74ahzpayhzhr2kn5mm75tevbnhhcoucxxq&#x40;reply.github.com&#x29; by wizyty&#x40;afya.pl from Praktyka Lekarska AFYA - POMOC. Please contact the sender&#x28;wizyty&#x40;afya.pl&#x29; for any clarifications.

**Comment by lah7** [comment]
Stale issue, to update 4+ years later:

* OpenRazer has a persistence feature for many years, which resolves this issue. ✅  Last settings are applied at login.
* If a custom effect is created within Polychromatic, this is automatically replayed at login too.
* There is no v1.0.0 yet. There were many smaller v0.x.x releases instead.
* In future, we'd like Polychromatic to have an interface to setup lighting events that happen at login or logout.
